containers/container.yml

```diff
@@ -76,10 +76,6 @@ services:
     from: centos:7
     roles:
       - squid
-  content-server:
-    from: foreman-pulp-base:latest
-    roles:
-      - content-server
```

Comment:
Same as before, can we remove this role?

Reply:
With fire -- done

Reply:
:fire: :fire: :scissors: :+1: